### ISSUE TITLE
Can't connect to any server

### ISSUE THREAD

**Issue by n00b660** [space-wizards/space-station-14#33208]
## Description
In the main tab all favorited servers are offline but in the servers tab they are not offline and have players. Attempting to connect to any server just ends up in failed to connect.
Don't really know what caused this to happen since yesterday everything was working.

**Reproduction**
Open launcher > Attempt to join server through server tab > Connection failed

**Screenshots**
<img width="601" alt="image" src="https://github.com/user-attachments/assets/35452c96-c7fc-4770-b3be-32cfdb9fb570">
<img width="586" alt="image" src="https://github.com/user-attachments/assets/5b0f866f-4ad4-49be-b326-f06004bc5426">
<img width="259" alt="image" src="https://github.com/user-attachments/assets/fe7d73b8-8f69-43ca-be18-0f2348af96a1">
<img width="254" alt="image" src="https://github.com/user-attachments/assets/04ae60a7-94b2-41e8-94cd-02f2fc0748c2">

**Additional context**
steam might be causing this problem since I can't actually launch the game from steam because it just does not launch and I'm forced to launch the game directly through it's files

**Comment by deltanedas** [comment]
are you sure the favorited ip didnt just change

**Comment by n00b660** [comment]
[launcher.log](https://github.com/user-attachments/files/17667224/launcher.log)
Couldn't  upload the log before but here it is.

**Comment by n00b660** [comment]
> are you sure the favorited ip didnt just change

Nope, 100% nothing changed and this is an issue with EVERY server.

**Comment by n00b660** [comment]
Also I tried reinstalling the game, rebooting PC, removing everything ss14 related in appdata and this didn't get fixed.

**Comment by n00b660** [comment]
Servers suddenly started working a few hours ago, could connect to every server but now once again the servers are offline, but this time not every server is offline, just 4
![image](https://github.com/user-attachments/assets/346266a9-2e17-41be-8b8e-a37083fb934f)

**Comment by n00b660** [comment]
nevermind, can't connect to any server at all and 
now the error message when trying to join servers is different
<img width="319" alt="image" src="https://github.com/user-attachments/assets/32d93b5d-0336-4ab8-be30-4a39d7f7905d">

**Comment by n00b660** [comment]
launcher also logged both the time when the servers were working and now
[launcher.log](https://github.com/user-attachments/files/17683402/launcher.log)

**Comment by PJB3005** [comment]
We've got other similar reports and it looks like Russia has just started blocking Hetzner infrastructure? God damnit man...

**Comment by n00b660** [comment]
Oh my god why does everything have to be blocked?? and I just started playing SS14 again after a long time...

**Comment by DocNITE** [comment]
Same problem.
